mindjunkies/dashboard/tests/test_mixin.py

```
@@ -0,0 +1,58 @@
+@override_settings(
+    ROOT_URLCONF="test_urls",
+    MIDDLEWARE=[
+        "django.middleware.security.SecurityMiddleware",
+        "django.contrib.sessions.middleware.SessionMiddleware",
+        "django.middleware.common.CommonMiddleware",
+        "django.middleware.csrf.CsrfViewMiddleware",
+        "django.contrib.auth.middleware.AuthenticationMiddleware",
+        "django.contrib.messages.middleware.MessageMiddleware",
+    ],
+)
+class TestCustomPermissionRequiredMixin(TestCase):
```

Ruff (0.8.2)

Undefined name `TestCase`

(F821)